### ISSUE TITLE
funasr里面没有Whisper的微调支持呀

### ISSUE THREAD

**Issue by moro0v0** [modelscope/FunASR#2678]
whisper的model.py文件里面forward函数只有一个pass

**Comment by LauraGPT** [comment]
FunASR 中 Whisper 的集成是作为推理模型使用的，不支持微调训练。

如需微调语音识别模型，推荐以下方案：

1. **Fun-ASR-Nano 微调** — 基于 LLM 架构，31 种语言，效果最好：
   - 训练文档：[FunASR Training Guide](https://modelscope.github.io/FunASR/training.html)

2. **SenseVoice 微调** — 非自回归，速度最快：
   - 训练脚本在 [SenseVoice 仓库](https://github.com/FunAudioLLM/SenseVoice) 的 `finetune.py`

3. **Paraformer 微调** — 中文生产级：
   - 同样参考 [Training Guide](https://modelscope.github.io/FunASR/training.html)

如果必须微调 Whisper 本身，建议使用 OpenAI 官方的 [whisper repo](https://github.com/openai/whisper) 或 HuggingFace transformers 的 Whisper 实现。

**Comment by LauraGPT** [comment]
已添加 Whisper 微调支持！

**修复内容**：
- 实现了 `WhisperWarp.forward()` 方法（之前只有 `pass`），计算标准 cross-entropy loss
- 添加了微调示例脚本 `examples/industrial_data_pretraining/whisper/finetune.sh`
- 添加了微调文档

**使用方式**：

1. 准备数据（JSONL 格式）：
```json
{"key": "utt1", "source": "/path/to/audio.wav", "target": "transcription text"}
```

2. 运行微调：
```bash
cd examples/industrial_data_pretraining/whisper
bash finetune.sh
```

支持所有 Whisper 模型（tiny 到 large-v3-turbo）。

请更新到最新代码：`git pull` 或等下个 PyPI 版本。
